### PR TITLE
feat: use accessible items on home

### LINKED ISSUE #794: Pagination: Own items list

### LINKED ISSUE #886: Show accessible items
First step to merge own + shared.

For now we display all accessible items from the My Graasp tab.

### PR DESCRIPTION
**Pull request by pyphilia**
Thanks to @LinaYahya previous work this PR was really easy to do. Instead of applying the pagination on own items, I've done it on another endpoint to avoid losing some features (copy, move, navigation that show the full list of own items).
In addition pagination and search are done in the backend (only for home page).

By default I've set `true` to show only items created by us, but probably it's better to set it to `false` to reveal the full potential of the page. What do you think @juancarlosfarah @dgillet @swouf @LinaYahya @spaenleh  ?

Screenshot 2023-12-01 at 17 01 25

Reminder: we have to merge own and shared items because some edge cases prevent you to find your items (an item you created but is not a root cannot be displayed)

TODO:
- [x] sorting
- [x] tests

close #886 
close #794

### CODE REVIEW

File: src/components/main/Home.tsx [hunk 1]
```diff
@@ -1,45 +1,134 @@
...
+  | 'item.name'
+  | 'item.type'
+  | 'item.created_at'
+  | 'item.updated_at';
+
 const HomeLoadableContent = (): JSX.Element => {
   const { t: translateBuilder } = useBuilderTranslation();
-  const { data: ownItems, isLoading, isError, isSuccess } = hooks.useOwnItems();
+  const { data: currentMember } = useCurrentUserContext();
+  const [showOnlyMe, setShowOnlyMe] = useState(false);
```
[the remark this answers is not in the document]
Comment (pyphilia) [reply src/components/main/Home.tsx:38]: What do you mean? a query param for builder or /accessible? For accessible it's `creatorId`

File: src/components/main/ItemsToolbar.tsx
[the remark this answers is not in the document]
Comment (pyphilia) [reply src/components/main/ItemsToolbar.tsx:1]: Indeed it's a bit too nested in the components. But if I want to place it where I want + not break the other tables I have to edit a lot of files. I think a better move is to do that while changing the tables.

**Review by spaenleh — APPROVED** [review]
🎅🏻
